docs/conf.py

```diff
@@ -31,52 +31,52 @@
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    'plantweb.directive',
-    'sphinx.ext.autodoc',
-    'sphinx.ext.coverage',
-    'sphinx.ext.doctest',
-    'sphinx.ext.githubpages',
-    'sphinx.ext.graphviz',
-    'sphinx.ext.ifconfig',
-    'sphinx.ext.intersphinx',
-    'sphinx.ext.todo',
-    'sphinx.ext.viewcode',
-    'sphinxcontrib.spelling',
+    "plantweb.directive",
+    "sphinx.ext.autodoc",
+    "sphinx.ext.coverage",
+    "sphinx.ext.doctest",
+    "sphinx.ext.githubpages",
+    "sphinx.ext.graphviz",
+    "sphinx.ext.ifconfig",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.todo",
+    "sphinx.ext.viewcode",
+    "sphinxcontrib.spelling",
 ]
 
 # Plantweb configuration
-plantweb_defaults = {
-    'format': 'png',
-}
+plantweb_defaults = {"format": "png"}
 
 # Add any paths that contain templates here, relative to this directory.
-templates_path = ['_templates']
+templates_path = ["_templates"]
 
 # The suffix(es) of source filenames.
 # You can specify multiple suffix as a list of string:
 #
 # source_suffix = ['.rst', '.md']
-source_suffix = ['.rst', '.md']
+source_suffix = [".rst", ".md"]
 
 # The master toctree document.
-master_doc = 'index'
+master_doc = "index"
 
 # General information about the project.
-project = 'RENKU'
-copyright = '2017-2019, Swiss Data Science Center'
-author = 'Eric Bouillet, Julien Eberle, Fotis Georgatos, ' \
-         'Jakub Chrobasik, Jiri Kuncar, Izabela Moise, ' \
-         'Rok Roskar, Sofiane Sarni, Sandra Savchenko De Jong, ' \
-         'Johann-Michael Thiebaut, Olivier Verscheure'
+project = "RENKU"
+copyright = "2017-2019, Swiss Data Science Center"
+author = (
+    "Eric Bouillet, Julien Eberle, Fotis Georgatos, "
+    "Jakub Chrobasik, Jiri Kuncar, Izabela Moise, "
+    "Rok Roskar, Sofiane Sarni, Sandra Savchenko De Jong, "
+    "Johann-Michael Thiebaut, Olivier Verscheure"
```

Comment:
Should this list be aligned with the one at line 172?

Reply:
Thanks I missed that there were two lists of authors... should be fixed now.